internal/organizations/organization_test.go

```diff
@@ -0,0 +1,176 @@
+package organizations
+
+import (
+	"bytes"
+	"testing"
+
+	root "github.com/equinix/metal-cli/internal/cli"
+	outputPkg "github.com/equinix/metal-cli/internal/outputs"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCli_OperatingSystem(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode.")
+	}
+	subCommand := "organization"
+	consumerToken := ""
+	apiURL := ""
+	Version := "metal"
+	rootClient := root.NewClient(consumerToken, apiURL, Version)
+	type fields struct {
+		MainCmd  *cobra.Command
+		Outputer outputPkg.Outputer
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		want    *cobra.Command
+		cmdFunc func(*testing.T, *cobra.Command)
+	}{
+		{
+			name: "get",
+			fields: fields{
+				MainCmd:  NewClient(rootClient, outputPkg.Outputer(&outputPkg.Standard{})).NewCommand(),
+				Outputer: outputPkg.Outputer(&outputPkg.Standard{}),
+			},
+			want: &cobra.Command{},
+			cmdFunc: func(t *testing.T, c *cobra.Command) {
+				t.Helper()
+				root := c.Root()
+				out := &bytes.Buffer{}
+				root.SetArgs([]string{subCommand, "get"})
+				root.SetOutput(out)
+				err := root.Execute()
+				assert.NoError(t, err)
+			},
+		},
+		{
+			name: "create",
+			fields: fields{
+				MainCmd:  NewClient(rootClient, outputPkg.Outputer(&outputPkg.Standard{})).NewCommand(),
+				Outputer: outputPkg.Outputer(&outputPkg.Standard{}),
+			},
+			want: &cobra.Command{},
+			cmdFunc: func(t *testing.T, c *cobra.Command) {
+				t.Helper()
+				root := c.Root()
+				out := &bytes.Buffer{}
+				root.SetArgs([]string{subCommand, "create", "-n", "test-org"})
+				root.SetOutput(out)
+				err := root.Execute()
+				assert.NoError(t, err)
+			},
+		},
+		{
+			name: "create",
+			fields: fields{
+				MainCmd:  NewClient(rootClient, outputPkg.Outputer(&outputPkg.Standard{})).NewCommand(),
+				Outputer: outputPkg.Outputer(&outputPkg.Standard{}),
+			},
+			want: &cobra.Command{},
+			cmdFunc: func(t *testing.T, c *cobra.Command) {
+				t.Helper()
+				root := c.Root()
+				out := &bytes.Buffer{}
+				root.SetArgs([]string{subCommand, "create", "-n", "test-org1", "-w", "www.metal.equinix.com"})
```

Comment:
In the terraform provider acceptance tests, we use a naming convention (`tfacc-<resource>-<suffix>`, e.g. `tfacc-org-asdlfkjlkj`) for all resources that are created by the tests so that we can easily identify those resources.  We should adopt a similar naming convention in this project, something like `cli-acc-<resource>-<suffix>`.

End-to-end tests should be written in such a way that resources created by the tests are cleaned up, but using a naming convention allows us to easily identify test resources if we need to clean them up separately for some reason.

```suggestion
				root.SetArgs([]string{subCommand, "create", "-n", "cli-acc-org-test1", "-w", "www.metal.equinix.com"})
```

Reply:
updated